packages/datatrak-web/src/layout/UserMenu/UserMenu.tsx

```diff
@@ -30,18 +30,19 @@ export const UserMenu = () => {
   const [menuOpen, setMenuOpen] = useState(false);
   const onCloseMenu = () => setMenuOpen(false);
   const toggleUserMenu = () => setMenuOpen(!menuOpen);
+  const isDesktop = !useIsMobile();
```

Comment:
Just to avoid the unnecessary negative
```suggestion
  const isDesktop = useIsDesktop();
```